pricesAPITypeScript/main.ts

```diff
@@ -0,0 +1,37 @@
+import moment from 'moment'
+import fetch from 'node-fetch'
+
```

Comment:
could u rename the file to `prices` in that ways is more readable, we already know is Typescript because the .ts extention